src/main/java/Aether.java

```diff
@@ -0,0 +1,206 @@
+    private Task[] tasks = new Task[MAX_TASKS];
+    private int taskCount = 0;
+
+    public static void main(String[] args) {
+        Aether aether = new Aether();
+        aether.run();
+    }
+
+    public void run() {
+        Display.showStartScreen();
+        Scanner scanner = new Scanner(System.in);
+
+        // Main loop to keep the program running until the user exits
+        while (!isExit) {
+            System.out.println("You:");
+            String command = scanner.nextLine();
+            Display.printSeparator();
+            handleCommand(command);
+        }
+    }
+
+    private void handleCommand(String command) {
+        int index = 0;
+
+        // Trim leading and trailing spaces and split the command into parts
+        command = command.trim();
+        String[] commandParts = command.split(" ", 2);
+        String commandName = commandParts[0].toLowerCase();
+
+        String arguments = commandParts.length > 1 ? commandParts[1] : "";
+
+        // Switch to Handle different commands
+        switch (commandName) {
+        case "bye":
+            isExit = true;
+            Display.showEndScreen();
+            break;
+        case "list":
+            listTasks();
+            break;
+        case "mark":
+            index = Integer.parseInt(arguments) - 1;
+            markTaskStatus(index, true);
+            break;
+        case "unmark":
+            index = Integer.parseInt(arguments) - 1;
+            markTaskStatus(index, false);
+            break;
+        case "todo":
+            addTask(new Todo(arguments));
+            break;
+        case "deadline":
+            String[] deadlineParts = arguments.split(" /by ", 2);
+            addTask(new Deadline(deadlineParts[0], deadlineParts[1]));
+            break;
+        case "event":
+            String[] eventParts = arguments.split(" /from | /to ");
+            addTask(new Event(eventParts[0], eventParts[1], eventParts[2]));
+            break;
+        default:
+            Display.response("Invalid command. Please try again.");
+            break;
+        }
+        Display.printSeparator();
+    }
+
+    private void addTask(Task task) {
+        if (taskCount < tasks.length) {
+            tasks[taskCount] = task;
+            taskCount++;
+            String taskWord = taskCount == 1 ? "task" : "tasks";
+            Display.response("Got it. I've added this task:\n  " + task +
+                    "\nNow you have " + taskCount + " " + taskWord + " in the list.");
+        } else {
+            Display.response("Task list is full. Sorry!");
+        }
+    }
+
+
+    private void listTasks() {
+        if (taskCount == 0) {
+            Display.response("Task list is empty.");
+        } else {
+            Display.response("Here are the tasks in your list:");
+            for (int i = 0; i < taskCount; i++) {
+                System.out.println((i + 1) + "." + tasks[i]);
+            }
+        }
+    }
```

Comment:
Loops and conditions wrapped appropriately using curly brackets even for single code blacks as per Java coding standard. This provides clarity to the reader and makes the code less bug-prone.